backup-and-restore.adoc

```diff
@@ -0,0 +1,130 @@
+
+This tutorial has been tested with link:https://github.com/kaleidos-ventures/taiga-docker[the official Taiga6 docker image].
+
+First, create a directory on the host where you want to store your backups in and switch into it. For this example we're going to use
+
+[source,bash]
+----
+:/# mkdir taiga-backup
+:/# cd taiga-backup
+----
+
+We have to find the container ID of the database container. To get a list of all running containers including their IDs, run
+
+[source,bash]
+----
+:/# docker ps -a
+----
+
+Search for the container with "taiga-docker_taiga-db" in the name and copy the ID.
+
+Export the database contents from the database container into your backup directory on your host machine via sql dump.
+
+[source,bash]
+----
+:/# docker exec "TAIGA-DB CONTAINER ID" pg_dump -U taiga taiga > taiga-db-backup.sql
+----
+
+Everybody will get a different container ID but to give a concrete example how this should look:
+
+[source,bash]
+----
+:/# docker exec f820941cbedc pg_dump -U taiga taiga > taiga-db-backup.sql
+----
+
+Search for the container with "taiga-docker_taiga-back_1" in the image name and copy the ID. 
+
+Then tar the media files inside the taiga-back container.
+
+[source,bash]
+----
+:/# docker exec "TAIGA-BACK CONTAINER ID" tar czf taiga-media-backup.tar.gz media
+----
+
+Copy them them out of the container into the taiga-backup directory on your host.
+
+[source,bash]
+----
+:/# docker cp "TAIGA-BACK CONTAINER ID":/taiga-back/taiga-media-backup.tar.gz .
+----
+
+Anyone should only need to backup the database and the media files but exclude the static files, cause they are recollected anytime with Django collectstatic.
+
+== Recommended restore process for Taiga6 docker
+
+This tutorial assumes that you have fresh server running and you want to restore your backed up Taiga6 data. You installed all the preriquisities, cloned the Taiga6 docker repository and moved into it.
+
+Follow these steps:
+[source,bash]
+----
+# Run only the database to avoid migrations
+$ docker-compose up -d taiga-db
+
+# Copy the dump inside the container:
+$ docker cp taiga-db-backup.sql taiga-docker_taiga-db_1:/taiga-db-backup.sql
+
+# Access the container
+$ docker exec -ti taiga-docker_taiga-db_1 /bin/bash
+----
+
+And inside the container, load the dump:
+[source,bash]
+----
+:/# psql -U taiga taiga < taiga-db-backup.sql
+----
+
+Check that the data have been properly migrated. Delete the taiga-db-backup.sql file and exit the container.
+
+=== Migrate the database
+
+Next step is to run Taiga6 backend so the database can be migrated to the new schema:
+[source,bash]
+----
+$ docker-compose up -d taiga-back
+----
+
+=== Migrate the media files
+
+With the backend up and running, copy the taiga5_media.tar.gz file inside the backend container and access the container:
```

Comment:
typo: it's not "taiga5_media" because we're backup'ing and restoring taiga6. Above, you called it 'taiga-media-backup.tar.gz`. We can stick with this name.

Reply:
Done ✅